pkgs/by-name/li/libmtp/package.nix

```diff
@@ -41,7 +56,7 @@ stdenv.mkDerivation rec {
 
   propagatedBuildInputs = [ libusb1 ];
 
-  preConfigure = "NOCONFIGURE=1 ./autogen.sh";
+  preConfigure = "autopoint -f; NOCONFIGURE=1 ./autogen.sh";
```

Comment:
Usually we'd write this on multiple lines…

Reply:
Sounds good. Switched to command per line